### ISSUE TITLE
Changing GUI Title

### ISSUE THREAD

**Issue by dylanlogann** [tqdm/tqdm#731]
Hi,

I have read the documentation, and looked at the back-end code for the tqdm_gui class, and I cannot find anything about setting the title of the window to 'Figure %i', is there any way to make the window title custom? So for instance I want it to be called 'Download Progress', instead of 'Figure 1' 'Figure 2' etc. I am using windows, and have read previous issues and found nothing.

**Comment by casperdcl** [comment]
ah, I should really update the submodules (`notebook` & `gui`) to be more compatible with core `tqdm` - haven't done so in ages.

**Comment by dylanlogann** [comment]
Thank you casper. will this update introduce custom GUI title functionality?

**Comment by casperdcl** [comment]
probably, I actually wrote the matplotlib gui as a sort of hack years ago. The notebook frontend also hasn't been updated properly for some time. Probably will introduce some customisation abilities. There's also things like #734 if i ever find the time

**Comment by dylanlogann** [comment]
@casperdcl 

yeah thats pretty cool, just keep us updated! I think its a really cool thing you've got going here!

**Comment by casperdcl** [comment]
Yes I need to update the code to follow my own documentation (https://github.com/tqdm/tqdm/#custom-integration) lol. 

[![](https://img.shields.io/badge/tips_raised-£22-ff69b4.svg?logo)](https://caspersci.uk.to/donate) not that great a thing :)
